cfgov/jinja2/v1/owning-a-home/closing-disclosure/index.html

```diff
@@ -0,0 +1,96 @@
+                                block__padded-top
+                                block__flush-bottom
+                                u-right-on-desktop">
+                        {{ social_media.render( {
+                        "twitter_text": "Use the @CFPB’s interactive tool to double-check all the details of your Closing Disclosure.",
+                        "email_title": "Get Closing Disclosure help from the CFPB",
+                        "email_text": "The CFPB’s tool helps you double-check your closing disclosure:",
+                        "email_signature": "-- From the CFPB",
+                        "linkedin_title": "Double-check your Closing Disclosure with the @CFPB",
+                        "linkedin_text": "Use this tool from the @CFPB to double-check that all the details in your closing disclosure are correct. Via @CFPB"
+                        } ) }}
+                    </div>
+                </div>
+                <div class="content-l_col-1-3">
+                    <img src="{{ static('img/ill-form-explainer-closing-disc.png')}}"
+                         alt=""
+                         class="illustration">
+                </div>
+            </div>
+            {# Import the form explainer macros #}
+            {% import "_templates/form-explainer.html" as form_explainer %}
+            {# Import the data to use with form explainer #}
+            {% from "closing-disclosure-1-data.html" import data as page_1 %}
+            {% from "closing-disclosure-2-data.html" import data as page_2 %}
+            {% from "closing-disclosure-3-data.html" import data as page_3 %}
+            {% from "closing-disclosure-4-data.html" import data as page_4 %}
+            {% from "closing-disclosure-5-data.html" import data as page_5 %}
+            {# Render form explainer with the data for this form #}
+            <div class="col-12">
+                {{ form_explainer.render([ page_1, page_2, page_3, page_4, page_5 ]) }}
+            </div>
+            <div class="col-12">
+                {% include "_templates/ratings-form.html" %}
+            </div>
+        </div>
+    </div>
+    {% import "_templates/brand-footer.html" as brand_footer %}
+    {% set footer_links = [ {
+        "img": static("img/footer_keys.png"),
+        "text": "Found a home, chosen a lender?",
+        "label": "Get ready to close",
+        "url": "/owning-a-home/process/close/"
+    }, {
+        "img": static("img/footer_checklist.png"),
+        "text": "Know what to expect during closing?",
+        "label": "Get our closing checklist",
+        "url": "/owning-a-home/resources/checklist_mortgage_closing.pdf",
+        "is_jump_link": true
+    }, {
+        "img": static("img/footer_closing-forms.png"),
+        "text": "Ready to review your closing documents?",
+        "label": "Learn what to look for",
+        "url": "/owning-a-home/resources/mortgage_closing_forms.pdf",
+        "is_jump_link": true
```

Comment:
Is there anywhere where `is_jump_link` is false?

Reply:
yeah, so the above code omitted the value because the default was false. I just added it and set
it to false.